### PR TITLE
Release/1.5.0

### PR DESCRIPTION
**Pull request by xaun**
@cesar-tonnoir not ready for merge, having troubles building this in impac-express.

error which I don't understand why: `ECONFLICT Unable to find suitable version for angular-cookies`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -33,8 +33,16 @@
     "lodash": "~3.10.1",
     "components-font-awesome": "~4.5.0",
     "angular-poller": "~0.4.5",
-    "moment": "~2.11.2",
-    "pusher": "^3.0.0"
+    "moment": "^2.11.2",
+    "angular-toastr": "^1.7.0",
```
Comment (alexnoox) [inline bower.json]: `angular-toaster` is already present in this file.

File: bower.json [hunk 2]
```diff
@@ -33,8 +33,16 @@
...
+    "pusher": "^3.0.0",
+    "angular-translate": "^2.11.0",
```
Comment (alexnoox) [inline bower.json]: Please set the version of `angular-translate` to `~2.13.0`

File: bower.json [hunk 3]
```diff
@@ -33,8 +33,16 @@
...
+    "angular-translate-handler-log": "^2.11.0",
+    "angular-translate-loader-static-files": "^2.11.0",
+    "angular-translate-storage-cookie": "^2.11.0",
+    "angular-translate-storage-local": "^2.11.0",
+    "angular-translate-interpolation-messageformat": "^2.11.0",
+    "angular-sanitize": "^1.5.7"
```
Comment (alexnoox) [inline bower.json]: Should be: `"angular-sanitize": "~1.6.0",`

File: bower.json [hunk 4]
```diff
@@ -33,8 +33,16 @@
...
     "angular-poller": "~0.4.5",
-    "moment": "~2.11.2",
-    "pusher": "^3.0.0"
+    "moment": "^2.11.2",
+    "angular-toastr": "^1.7.0",
+    "pusher": "^3.0.0",
+    "angular-translate": "^2.11.0",
+    "angular-translate-handler-log": "^2.11.0",
+    "angular-translate-loader-static-files": "^2.11.0",
+    "angular-translate-storage-cookie": "^2.11.0",
```
Comment (alexnoox) [inline bower.json]: Shouldn't this dependency be passed in `devDependencies` as the host frontend project will be the one setting the `$translate.use()` and managing storing strategy? (eg, in mnoe-angular the locale key to use it in the uri)

Comment (alexnoox) [inline bower.json]: BTW, this dependency is the one introducing the `ECONFLICT Unable to find suitable version for angular-cookies`.

Comment (xaun) [inline bower.json]: @alexnoox I'm not 100% sure, the strategy currently seems to be managed in impac-angular.config.coffee. Can you please take a look at this file, should that be happening there? @clemthenem added that I think and I'm not too sure what these strategy's are doing.

**Comment by xaun** [comment]
This is not merged, It has ben removed from release/1.5.0 due to postponement - see feature/internationalization for latest.

**Pull request merged**